app/controllers/api/base_controller/parameters.rb

```diff
@@ -161,6 +161,8 @@ def sort_params(klass)
         options = String(params['sort_options']).split(",")
         params['sort_by'].split(",").zip(orders).collect do |attr, order|
           if klass.attribute_method?(attr) || klass.method_defined?(attr) || attr == klass.primary_key
+            raise BadRequestError,
+                  "#{klass.name} cannot be sorted by #{attr}" unless klass.attribute_supported_by_sql?(attr)
```

Comment:
@imtayadeway no it's not implied on 163. I will try to simplify it, but I also want to send back that specific error message, rather than the one below that indicates it is not an attribute (since it is, just not a sortable one)